src/App.js

```diff
@@ -1,16 +1,45 @@
 import React from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
+import { useState} from 'react';
 
 const App = () => {
+  // I need an array that has all of my state
+  // const initialCopy = chatMessages.map(chat=> {
+  //   return {...chat,
+  const [initialCopy, setChatEntries] = useState(chatMessages);
```

Comment:
Great job using state for the chat messages! I'd recommend a different variable name for state here though. `initialCopy` doesn't give us any information on what kind of data it is holding. Also, since state is meant to be updated, this variable won't always hold the initial copy of the chat messages. Remember that good variable names should let developers know what data it contains at a glance.